src/Forms/MultiSelectField.php

```diff
@@ -72,11 +78,28 @@ public function setValue($value, $obj = null)
         if ($obj instanceof DataObject) {
             $this->loadFrom($obj);
         } else {
-            parent::setValue($value);
+            parent::setValue($value, $obj);
         }
         return $this;
     }
 
+    public function setSubmittedValue($value, $data = null)
+    {
+        // When no value is selected and a regular edit form is submitted,
+        // usually an empty string will be POSTed e.g. MyField[]: ""
```

Comment:
```suggestion
        // usually an empty string will be POSTed e.g. `MyField[]: ""`
```
Just making it a little clearer that that's all part of the same code example.

Reply:
This was updated